source/features/useful-not-found-page.tsx

```diff
@@ -169,10 +160,20 @@ async function initPRCommit(): Promise<void | false> {
 	);
 }
 
-void features.add(__filebasename, {
-	include: [
+void features.add(__filebasename, 	{
+	always: [
 		pageDetect.is404,
 	],
+	include: [() => parseCurrentURL().length > 1],
```

Comment:
Since both are required

```suggestion
	always: [
		pageDetect.is404,
		() => parseCurrentURL().length > 1,
	],
```